### PR TITLE
REPAIRS THE GOLDEN THRONE

### PR DESCRIPTION
**Pull request by clusterfack**
ALL HAIL THE EMPEROR

### CODE REVIEW

File: code/ZAS/ConnectionGroup.dm [hunk 1]
```diff
@@ -129,7 +129,7 @@ Class Procs:
 							continue
 					else if(!(M.loc in src:A.contents))
 						continue
-						M.GotoAirflowDest(differential/10)
+					M.GotoAirflowDest(differential/10)
```
Comment (kol1th) [inline code/ZAS/ConnectionGroup.dm:132]: Did you even test this?

**Comment by Skullyton** [comment]
???

**Comment by Exxion** [comment]
Memes aside, this fixes airflow not pulling things

**Comment by Skullyton** [comment]
!!!

**Comment by Exxion** [comment]
Record-breaking pace

**Comment by clusterfack** [comment]
THANK THE EMPEROR FOR THE SKILLS OF OUR TECH PRIESTS.

**Comment by kol1th** [comment]
image

**Comment by troots** [comment]
what

**Comment by jakmak** [comment]
Hwat

**Comment by 9600bauds** [comment]
Post webms of the now-restored golden throne
https://dl.dropboxusercontent.com/u/29768571/Desktop%2001.04.2017%20-%2000.49.43.03.DVR.webm

**Pull request merged**